transactional_licence_form/templates/start3.html

```diff
@@ -1,5 +1,351 @@
+                </span>
+              </div>
+              <div class="start-form">
+                <strong>Do you intend to apply for a licence?</strong>
+                <p>
+                  If you intend to use computational analysis on Find Case Law records, you will need a computational analysis licence
+                </p>
+                <ol>
+                  <li>Do you intend to use a computational analysis on Find Case Law records?</li>
+                </ol>
+                <strong>Contact details</strong>
+                <p>Full name of person we can discuss your application with</p>
+                <p>Email address of person we can discuss your application with</p>
+                <p style="text-indent: 1rem;">2a. Full name of person responsible for licence compliance</p>
+                <p style="text-indent: 1rem;">3a. Email address of person responsible for licence compliance</p>
+                <strong>About your organisation</strong>
+                <ol start="4">
+                  <li>What is the full legal name of your organisation?</li>
+                  <li>Is your organisation known by any other names?</li>
+                  <li>Where is your organisation registered?</li>
+                  <li>
+                    What type of organisation is it?
+                    <br />
+                    Select all that apply
+                    <ul>
+                      <li>Private limited company</li>
+                      <li>Public limited company</li>
+                      <li>Partnership</li>
+                      <li>Sole trader</li>
+                      <li>Registered charity</li>
+                      <li>Community interesr company</li>
+                      <li>Independant research organisation</li>
+                      <li>Public body</li>
+                      <li>Independant body</li>
+                      <li>Other (please specify)</li>
+                    </ul>
+                  </li>
+                  <li>Please give your organisation identifier (e.g. company number or charity registration number)</li>
+                  <li>Please give the name of any partners or organisations you are working with</li>
+                </ol>
+                <strong>Purpose and Activities</strong>
+                <p>Your answers to this section may be published alongside your statement in the future.</p>
+                <ol start="10">
+                  <li>Please give any project or product name associated with this work</li>
+                </ol>
+              </div>
+            </div>
+          </details>
+          <a role="button"
+             tabindex="0"
+             draggable="false"
+             class="govuk-button"
+             data-module="govuk-button"
+             value="I want to perform computational analysis"
+             href="{% url 'transactional-licence-form-page-2' %}">Download as PDF</a>
```

Comment:
The URL here is wrong, but i suspect that's because we don't have the PDF yet? just noting down so we remember to change it, and to get the pdf!